src/app/f2f/controllers/checkDetails.js

```diff
@@ -127,17 +127,22 @@ class CheckDetailsController extends DateController {
       req.sessionModel.set("idHasExpiryDate", idHasExpiryDate)
       req.sessionModel.set("expiryDate", expiryDate);
       req.sessionModel.set("addressCheck", address);
+      
       //Confirmation display values
-      const idChoice = req.sessionModel.get("selectedDocument");
+      const idChoice = req.sessionModel.get("photoIdChoice");
+      const idTranslationKey = res.locals.translate(`photoIdChoice.items.${idChoice}.label`)
       const changeUrl = req.sessionModel.get("changeUrl");
       const addressCheck = req.sessionModel.get("addressCheck");
+      const addressCheckTranslationKey = res.locals.translate(`${idChoice}AddressCheck.items.${addressCheck}.label`)
       const hasExpiryDate = req.sessionModel.get("idHasExpiryDate");
+      const hasExpirydateTranslationKey = res.locals.translate(`idHasExpiryDate.items.${hasExpiryDate}.label`)
 
       locals.country = req.sessionModel.get("country");
       locals.formattedExpiryDate = formatDate(expiryDate, "YYYY-MM-DD");
-      locals.idChoice = idChoice;
+      locals.idTranslationKey = idTranslationKey
+      locals.addressCheckTranslationKey = addressCheckTranslationKey
+      locals.hasExpirydateTranslationKey = hasExpirydateTranslationKey
```

Comment:
Do you think we could add a couple of tests for these new additions?

Reply:
Absolutely!

Reply:
Resolving as this seems impossible with the current set up. Creating a tech debt ticket for us to start snapshot testing